modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala

```diff
@@ -33,11 +33,16 @@ object Smithy4sCodegenPlugin extends AutoPlugin {
     val smithy4sVersion =
       settingKey[String]("Smithy4sVersion")
 
+    @deprecated("Deprecated in 0.17, use `smithy4sInputDirs` instead.")
```

Comment:
I'm not sure if we need to deprecate it given we're about to break BC. Might as well just remove it, breaking plugin (only for the users who have a custom value here). Those who have an explicit smithy4sInputDir can migrate very easily.

Reply:
Yeah I'm cool with that personally. What say you @Baccata?

Reply:
Yeah, happy to remove